widget/playground/src/constants/languages.ts

```diff
@@ -32,4 +52,25 @@ export const LANGUAGES = [
   { name: 'Greek', value: 'el', Icon: Greece },
   { name: 'Italian', value: 'it', Icon: Italian },
   { name: 'Polish', value: 'pl', Icon: Poland },
+  { name: 'Afrikaans', value: 'af', Icon: SouthAfrica },
+  { name: 'Arabic', value: 'ar', Icon: SaudiArabia },
+  { name: 'Bengali', value: 'bn', Icon: Bengali },
+  { name: 'Catalan', value: 'ca', Icon: Catalonia },
+  { name: 'Danish', value: 'da', Icon: Denmark },
+  { name: 'Hindi', value: 'hi', Icon: India },
+  { name: 'Hungarian', value: 'hu', Icon: Hungary },
+  { name: 'Indonesian', value: 'id', Icon: Indonesian },
+  { name: 'Korean', value: 'ko', Icon: Korea },
+  { name: 'Lithuanian', value: 'lt', Icon: Lithuania },
+  { name: 'Malay', value: 'ms', Icon: Malay },
+  { name: 'Filipino', value: 'fil', Icon: Philippines },
+  { name: 'Serbian', value: 'sr', Icon: Serbia },
+  { name: 'Slovak', value: 'sk', Icon: Slovakia },
+  { name: 'Swahili', value: 'sw', Icon: Swahili },
+  { name: 'Thai', value: 'th', Icon: Thai },
+  { name: 'Turkish', value: 'tr', Icon: Turkish },
+  { name: 'Urdu', value: 'ur', Icon: Pakistan },
+  { name: 'Vietnamese', value: 'vi', Icon: Vietnamese },
+  { name: 'Greek(cyprus)', value: 'el-CY', Icon: Cyprus },
+  { name: 'Chinese(Traditional)', value: 'zh-TW', Icon: Chinese },
```

Comment:
```suggestion
  { name: 'Greek', value: 'el-CY', Icon: Cyprus },
  { name: 'Chinese (Traditional)', value: 'zh-TW', Icon: Chinese },
```

Reply:
We have two separate Greek languages, so they are named this way to distinguish them